pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[tool.poetry]
+name = "octopoes"
+version = "0.1.0"
```

Comment:
1.3?
Also add maintainer@openkat.nl as author.

Reply:
It was also 0.1.0 in the previous releases. It's mainly used for installing octopoes in boefjes and rocky. If we want to fix this  I ithink it would be better to properly look into this for the next release and fix it in all projects.